### ISSUE TITLE
`numeric-separators-style`: Allow onlyIfContainsSeparator with group size

### ISSUE THREAD

**Issue by Harpush** [sindresorhus/eslint-plugin-unicorn#2490]
### Description

Require separators only for X digits and higher. For lower amount of digits check only in contains separators.

### Fail

```js
10_24
10450
1_0451
```

### Pass

```js
1024
1_024
10_450
10_451
```

### Additional Info

The examamples are when 4 digits don't require separators but if exists they should be valid. 5 and above digits require separators and should be valid of course.

**Comment by sindresorhus** [comment]
Our rules are focused on consistency and this enables the opposite, so we are unfortunately going to pass on this.
